wazo_confd/plugins/context/resource.py

```diff
@@ -14,13 +15,29 @@
 class ContextList(ListResource):
     model = Context
     schema = ContextSchema
+    context_name_fmt = 'ctx-{tenant_slug}-{context_uuid}'
```

Comment:
Could you add the context type or an abbreviation for the type after the slug?
`ctx-{tenant_slug}-{context_type}-{context-uuid}'

The result would look like
`ctx-customer1-internal-1d07eb9a-0389-46da-9800-2d95dbce1e7e`
`ctx-customer1-incall-1d07eb9a-0389-46da-9800-2d95dbce1e7d`
`ctx-customer1-outcall-1d07eb9a-0389-46da-9800-2d95dbce1e7c`

Reply:
fixed